scripts/billing/sql/previous_day_costs_by_service.sql

```diff
@@ -0,0 +1,12 @@
+/*
+Query returns the previous day's total costs by service
+*/
+
+SELECT
+  service.id,
```

Comment:
We should probably alias the id and description fields as done in other files within this PR.

```sql
  service.id AS service_id,
  service.description AS service_description
```